src/js/experimental/media-chrome-listbox.js

```diff
@@ -12,8 +12,8 @@ template.innerHTML = `
     gap: 0.5em;
     margin: 0;
     padding: 0.5em;
-    background-color: var(--media-listbox-background, var(--media-control-background, rgba(10,10,15, .8)));
-    color: var(--media-text-color, white);
+    background: var(--media-control-background, var(--media-secondary-color, rgb(20 20 30 / 0.7)));
+    color: var(--media-control-font-color, var(--media-primary-color, rgb(238 238 238)));
```

Comment:
- Renaming `--media-text-color` to `--media-controll-font-color` for consistency and replacing with consistent composition, including defaulting (which again changes the default value)

Reply:
👍

Reply:
`font-color`, that feels off to me. a font:

> a file or set of characters (with a very specific set of characteristics, including weight, style, and width) used to create text in a specific style

it's the text that is colored, not the design of the letters.